### ISSUE TITLE
Show data set bounding boxes in 3D viewer

### ISSUE THREAD

**Issue by hanslovsky** [saalfeldlab/paintera#259]
It can be helpful to see the actual bounding box of open datasets in the 3D viewer.

**Comment by igorpisarev** [comment]
This should be easy when multi-resolution meshes are ready: I added an option to visualize bounding boxes of rendered blocks which helps to see what level of detail a particular block is rendered at.

**Comment by igorpisarev** [comment]
For visualizing the blocks I used `PolygonMesh` from FXyz library which allows to define a box as a set of quadrilateral faces. In contrast to JavaFX's standard `Box` primitive which is based on triangular faces, it doesn't have these unwanted diagonals in each face if you visualize it using the LINE mode.

**Comment by hanslovsky** [comment]
I'll check it out!
There is one caveat:
> Since release 0.4.0, FXyz3D-Client requires Java 9+ (for Java 8, use version 0.3.0)

https://github.com/FXyz/FXyz/releases/tag/0.4.0

**Comment by hanslovsky** [comment]
I stand corrected, the core library can be used with Java 8 in version 0.4.0 but for version 0.5.0 and later we would need a more recent Java:
https://github.com/FXyz/FXyz/releases/tag/0.5.0

**Comment by igorpisarev** [comment]
That's true, I learned it the hard way :) didn't see that remark in their release summary. Yes, I'm using 0.4.0 core library for these purposes and it works with Java 8.

If it's not an urgency, I'd suggest to postpone adding this feature until the multi-resolution meshes are merged to avoid conflicts and re-use the new code. (I'll create the PR soon)

**Comment by hanslovsky** [comment]
I will wait until the adaptive meshes are merged, then. I have some code that does show the bounding box successfully now, but I'll just c&p the relevant parts lateron.
